client/components/tinymce/plugins/simple-payments/dialog/index.jsx

```diff
@@ -219,11 +221,15 @@ class SimplePaymentsDialog extends Component {
 					.addPost( productToCustomPost( productForm ) )
 					.then( newProduct => {
 						dispatch( receiveUpdateProduct( siteId, customPostToProduct( newProduct ) ) );
+						analytics.tracks.recordEvent( 'calypso_simple_payments_button_create', {
+							price: productForm.price,
+							currency: productForm.currency,
+						} );
 						return newProduct.ID;
 					} );
 			} );
 		}
-
+		analytics.tracks.recordEvent( 'calypso_simple_payments_button_insert' );
```

Comment:
I think we do want to track when any button is inserted. `calypso_simple_payments_button_create` should get us stats on creating buttons